ios/Classes/PluginMethod.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PluginMethod.swift
+//
+//  Created by Kymer Gryson on 02/10/2019.
+//
+
+import Foundation
+
+enum PluginMethod {
```

Comment:
YASS 👍  but these still seem to be unused in ios/Classes/SwiftFlutterAdobeExperiencePlatformPlugin.swift ?

Reply:
Indeed, that's now fixed ; thanks for pointing that out :-)